apps/api/tsconfig.json

```diff
@@ -1,15 +1,15 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "outDir": "../../dist/libs/api",
```

Comment:
Can you please double-check that this is the intended behavior? This will export JS files in the top `/dist/libs/api` folder. It breaks the package boundaries of the monorepo and I am pretty sure our Docker setup won't be happy about it.